src/rules/maxClassesPerFileRule.ts

```diff
@@ -29,20 +36,25 @@ export class Rule extends Lint.Rules.AbstractRule {
         rationale: Lint.Utils.dedent`
             Ensures that files have a single responsibility so that that classes each exist in their own files`,
         optionsDescription: Lint.Utils.dedent`
-            The one required argument is an integer indicating the maximum number of classes that can appear in a file.`,
+            The one required argument is an integer indicating the maximum number of classes that can appear in a
+            file. An optional argument \`"include-class-expressions"\` can be provided to include class expressions
```

Comment:
better use `"exclude-class-expressions"` since that will not change the default behavior